src/api/v1/community_businesses/__tests__/register.test.integration.ts

```diff
@@ -0,0 +1,118 @@
+import * as Hapi from 'hapi';
+import * as Knex from 'knex';
+import { init } from '../../../../server';
+import { getConfig } from '../../../../../config';
+import { Organisation, Organisations, User, Users, CommunityBusinesses } from '../../../../models';
+import { getTrx } from '../../../../../tests/utils/database';
+import { StandardCredentials } from '../../../../auth/strategies/standard';
+import { RegionEnum, SectorEnum } from '../../../../models/types';
+import { Roles, RoleEnum } from '../../../../auth';
+
+
+describe('PUT /community-businesses', () => {
+  let server: Hapi.Server;
+  let knex: Knex;
+  let trx: Knex.Transaction;
+
+  let admin: User;
+  let adminCreds: Hapi.AuthCredentials;
+  let organisation: Organisation;
+  const config = getConfig(process.env.NODE_ENV);
+
+  beforeAll(async () => {
+    server = await init(config);
+    knex = server.app.knex;
+
+    server.app.EmailService = {
```

Comment:
Fancy setting up a simple mock of the `EmailService`? I think it would speed up the runtime of some of the tests.

Something like:
```ts
import MockEmailService from '../../tests/mocks/EmailService';

describe('', () => {
  let server: Hapi.Server;

  beforeAll(async () => {
    server = await init(config);
    MockEmailService.mock(server);
  });

  afterAll(async () => {
    MockEmailService.restore(server);
  });
})
```

Reply:
Are you suggesting moving L27-L40 to another file? Or to create a jest mock function?

I wonder if we could initalise the server in the global config n do something like `jest.mock('server.app.EmailService')`?

Unsure how this will speed up tests though, lemme know what you think n I can look into it further.